### PR TITLE
Another way to fix  Issue #2253

### PR DESCRIPTION
**Pull request by ubaidisaev**
before
after

### CODE REVIEW

File: web/src/components/pages/profile/layout.css [hunk 1]
```diff
@@ -74,10 +75,9 @@
             @media (--md-up) {
                 flex-direction: column;
                 margin: 0 auto;
-                width: 100%;
+                width: auto;
                 max-width: var(--wide-desktop-width);
                 position: absolute;
-                left: 0;
```
Comment (ubaidisaev) [inline web/src/components/pages/profile/layout.css:80]: here we need to apply `left` or `right` based on language `dir`.

File: web/src/components/pages/profile/layout.css [hunk 2]
```diff
@@ -74,10 +75,9 @@
...
                 right: 0;
             }
```
Comment (ubaidisaev) [inline web/src/components/pages/profile/layout.css:82]: here we need to apply `left` or `right` based on language `dir`.

**Comment by phirework** [comment]
Hi there, I'm not able to replicate that old UI bug anymore on the current version of CV, can you tell me what your steps to reproduce were and what your browser/OS/screen size specs are? https://commonvoice.mozilla.org/

**Comment by ubaidisaev** [comment]
> Hi there, I'm not able to replicate that old UI bug anymore on the current version of CV, can you tell me what your steps to reproduce were and what your browser/OS/screen size specs are? https://commonvoice.mozilla.org/

Hi. In resolution `~ 2000px` and higher. 
`Winddow  10`
`Chrome-86`
`firefox-82`

**Comment by phirework** [comment]
Gotcha. Instead of adding custom LTR/RTL rules I would actually prefer to refactor the column setup of the profile view to not rely on `--var(--wide-desktop-width)` for positioning. Is that something you'd feel comfortable taking on? You'd need to look at `layout.tsx` as well.

**Comment by ubaidisaev** [comment]
Fixed without relying on LTR/RTL rules.

**Comment by phirework** [comment]
Perfect, thank you!

**Pull request merged**